CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.17.0-dev.0
+TODO: update changelog
```

Comment:
What should the CHANGELOG look like? Should I remove the TODO? Or generate the changelog (in which case we should update instructions in https://github.com/flutter/devtools/tree/master/tool#prepare-the-release)

Reply:
I think TODO is fine for dev release. The generate changelog tool isn't smart enough to use releases that aren't formally tagged on our repo as a 'before' data for the range of commits. Plus, the release will be going out in a couple of days so this changelog will get updated then.

Reply:
perhaps we should just leave a bullet mentioning it is a dev release, so there aren't any notes needed?
Leaving todo's implies that something needs to be done and might come up later

Reply:
Added that to https://github.com/flutter/devtools/pull/4536 :)